GitCommands/Git/GitModule.cs

```diff
@@ -3337,6 +3336,8 @@ public GitBlame Blame(string? fileName, string from, Encoding encoding, string?
             }
         }
 
+        private static readonly Regex _headerRegex = new(@"^(?<objectid>[0-9a-f]{40}) (?<origlinenum>\d+) (?<finallinenum>\d+)", RegexOptions.Compiled);
```

Comment:
Please move this to the top of the file

Reply:
@pmiossec, could you move it before leaving yet?